File: my-code/448/main.go
package main

func findDisappearedNumbers(nums []int) []int {
	length := len(nums)
	for _, item := range nums {
		nums[(item-1)%length] += length
	}
	var ret []int
	for i, num := range nums {
		if num <= length {
			ret = append(ret, i+1)
		}
	}
	return ret
}

File: my-code/448/main.py
# -*- coding:utf-8 -*-
# @Time     : 2021/05/28 18: 09
# @Author   : Ranshi
# @File     : 448. 找到所有数组中消失的数字.py
from typing import List


class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        c = set(nums)
        return [i for i in range(1, len(nums) + 1) if i not in c]


if __name__ == "__main__":
    s = Solution()
    print(s.findDisappearedNumbers([1, 1]))
